invenio_circulation/pidstore/pids.py

```diff
@@ -17,5 +17,8 @@
 CIRCULATION_LOAN_FETCHER = 'loanid'
 """Fetcher PID for Loans."""
 
+CIRCULATION_LOAN_PID_ALIAS = 'loan_pid'
```

Comment:
Is this used here? How are you thinking of using this?

Reply:
No, it's not used in circulation but it will be used in ILS when referencing a `loan_pid` from another record.